src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -263,4 +272,37 @@ void GIVEN_customCAConfigurationWithACAChain_WHEN_registeringCAWithIotCore_THEN_
         assertEquals(lastRequest.coreDeviceCertificates(), expectedPem);
     }
 
+    @Test
+    void GIVEN_managedCAConfiguration_WHEN_updatedToCustomCAConfiguration_THEN_serverCertificatesAreRotated() throws
```

Comment:
This is the brand new test that ensures we are rotating the server certificate when CA configuration changes